tests/tasks/setup_candlepin.yml

```diff
@@ -53,6 +53,20 @@
         name: tomcat
         enabled: true
 
+    - name: Ensure /etc/pki/product exists
+      file:
+        path: /etc/pki/product
+        state: directory
+        mode: 0755
+
+    - name: Copy product certificates
+      copy:
+        src: "/home/candlepin/devel/candlepin/generated_certs/{{ item }}.pem"
+        dest: "/etc/pki/product/{{ item }}.pem"
+        remote_src: true
```

Comment:
you need a `mode` here to fix the ansible-lint error

Does installing candlepin generate some sort of default certs?

Reply:
> you need a `mode` here to fix the ansible-lint error

ack, added, thanks!

btw: should those lints be run automatically for PRs?

> Does installing candlepin generate some sort of default certs?

The product certificates of the test data are left in the sources, which is where they are copied from (only one for now).

Reply:
> > you need a `mode` here to fix the ansible-lint error
> 
> ack, added, thanks!
> 
> btw: should those lints be run automatically for PRs?

Yes.  They are run automatically.

> 
> > Does installing candlepin generate some sort of default certs?
> 
> The product certificates of the test data are left in the sources, which is where they are copied from (only one for now).

ok